### DISCUSSION TITLE
pretrained file

### DISCUSSION THREAD

**Discussion by Phelan164** [ultralytics/yolov5#2198] [Q&A] [answered]
What different between 2 pretrained file: yolov5x.pt and yolov5x**6**.pt

**Comment by glenn-jocher** [comment, marked as answer]
@Phelan164 YOLOv5x6 has an additional P6 output layer (4 outputs total, P3-P6), vs 3 outputs (P3-P5) in the standard models. The P6 layer is for larger objects, so we've trained this model at --img 1280. It performs at about 54.0 AP on COCO.

**Reply by Phelan164** [reply]
thanks @glenn-jocher
